src/CommunityStore/Product/ProductList.php

```diff
@@ -306,6 +312,9 @@ public function finalizeQuery(\Doctrine\DBAL\Query\QueryBuilder $query)
         if ($this->featuredOnly) {
             $query->andWhere("pFeatured = 1");
         }
+        if ($this->FilterManufacturers) {
+            $query->andWhere("pManufacturer =  $this->FilterManufacturers " );
```

Comment:
This is best handled with a parameterized query, so:
`$query->andWhere("pManufacturer = ?")->setParameter($paramcount++, $this->manufacturer);`